src/app/report/edit-display/edit-display.component.html

```diff
@@ -66,16 +66,14 @@
           (click)="openDifferenceModal('save')"
         ><i class="fa fa-save"></i></button>
       }
-      @if (report.encoding === 'Base64') {
-        <button class="btn btn-info" title="Convert to Base64" (click)="changeEncoding($event)">Base64</button>
-      }
+      <app-encoding-button [selectedNode]="selectedNode" (updatedMessageEvent)="setNewReport($event)"></app-encoding-button>
 
-      @if (report.stub) {
+      @if (selectedNode && selectedNode.stub) {
```

Comment:
This should only be visible if report is a checkpoint

Reply:
Added the change.